<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FmSwitchOriginalAndTranslation</class>
 <widget class="QDialog" name="FmSwitchOriginalAndTranslation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Switch original and translation</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="grpOriginal">
       <property name="title">
        <string>Location of new original</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="edOrigFile"/>
        </item>
        <item>
         <widget class="QToolButton" name="btBrowseFile">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="grpComments">
         <property name="title">
          <string>Original’s comments were taken from…</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radioOrig">
            <property name="text">
             <string>Author’s (leave intact)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioOrigTrans">
            <property name="text">
             <string>Author’s → translator’s</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioTrans">
            <property name="text">
             <string>Translator’s (also switch)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioTransOrig">
            <property name="text">
             <string>Translator’s → author’s</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
